### ISSUE TITLE
Test yellow background vs link text on focussed buttons

### ISSUE THREAD

**Issue by sastels** [cds-snc/report-a-cybercrime#814]

**Comment by sastels** [comment]
Fails WCAG AA

Probably the best solution is to slightly darken the blue until it meets the standard. In the app the links appear with white, yellow, or light grey backgrounds, so darkening them shouldn't cause something else to fail.

From https://webaim.org/resources/contrastchecker/
> WCAG 2.0 level AA requires a contrast ratio of at least 4.5:1 for normal text and 3:1 for large text. 
> WCAG Level AAA requires a contrast ratio of at least 7:1 for normal text and 4.5:1 for large text.

link text color: #005ea5
focus color: #ffbf47

![image](https://user-images.githubusercontent.com/8228248/67315486-afbaa680-f4d4-11e9-82f7-c5138836f4f9.png)
